Dockerfile

```diff
@@ -1,21 +1,48 @@
-FROM ubuntu:22.04 as tomcat
+FROM tomcat:9.0.85-jdk11-temurin-jammy
```

Comment:
```suggestion
FROM tomcat:9.0.86-jdk11-temurin-jammy
```

Reply:
Unfortunately, `tomcat:9.0.86` is not yet available as an official docker image. So I switched to `tomcat:9.0.85-jdk11-temurin-jammy` again.

As soon as a dependency update such as renovate (https://github.com/geoserver/docker/pull/48) is properly configured, this should be updated automatically / proposed as a PR.